### ISSUE TITLE
New unit: Gal

### ISSUE THREAD

**Issue by krofik** [nholthaus/units#272]
Gal is a unit for acceleration which is widely used in gravimetry / geodesy research. 1Gal == 1 cm / s^2, but also be defined as fraction of standard gravity. 
```c++
UNIT_ADD(acceleration , gal, Gals, Gal, unit<std::ratio<25493LL, 25000000LL>, standard_gravity>)
```
It would collapse with abbreviation of `Gallon` unit if lower case `Gal` ware used instead. 
Here is [wiki page](https://en.wikipedia.org/wiki/Gal_(unit)) and Germany's National Institute [reference ](https://www.ptb.de/cms/fileadmin/internet/Themenrundgaenge/ImWeltweitenNetzDerMetrologie/si.pdf) (page 30) about Gal.

**Comment by johelegp** [comment]
Why choose the fraction of standard gravity instead of 1 cm / s^2, when the latter is less likely to cause problems with precision and overflows?

**Comment by krofik** [comment]
You are right. Previously I had difficulties with defining compound units and therefore this solution looked plausible for me.

**Comment by nholthaus** [comment]
added support in 3.0. Consider backporting to 2.3

**Comment by nholthaus** [comment]
Added in the 3.x line: `acceleration.h` defines `gals` with abbreviation `Gal` as 1 cm/s^2 (`compound_conversion_factor<centimeters_, inverse<squared<seconds_>>>`).
